tests_metricflow/snapshots/test_custom_granularity.py/SqlPlan/DuckDB/test_join_to_time_spine_metric_grouped_by_custom_grain__plan0.sql

```diff
@@ -18,7 +18,7 @@ FROM (
     FROM (
       -- Change Column Aliases
       SELECT
-        subq_5.ds__day AS metric_time__day
+        subq_5.ds__day
```

Comment:
This change is due to the `join_on_time_dimension` being not longer required in the output of JoinToTimeSpineNode. It does not impact optimized SQL.